pkgs/test/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.9.4
+
+* Depend on the latest `package:test_core`.
```

Comment:
I think we should copy in the relevant change. This is the changelog users will see first, we shouldn't make them jump through extra hoops to figure out what "latest" means, then find the changelog for that version number in 2 other packages to see what changed.

We've done the same with `package:matcher` rolls in the past - we copy/paste the interesting changelog bits here.